.github/dependabot.yml

```diff
@@ -9,3 +9,9 @@ updates:
   allow:
   - dependency-type: direct
   - dependency-type: indirect
+- package-ecosystem: github-actions
+  directory: /
```

Comment:
@boahc077 can you clarify what this change means please?

Reply:
@nicoddemus This repository uses GitHub Actions workflows (defined here https://github.com/pytest-dev/pytest/tree/main/.github/workflows). Just Python pip packages, GitHub Actions are also updated on a periodic basis. This PR will enable dependabot to upgrade versions for GitHub Actions used by this repo similar to how it processes pip package upgrades currenty.

Reply:
Thanks!